eth_tester/backends/pyevm/main.py

```diff
@@ -206,6 +207,11 @@ def _get_vm_for_block_number(chain, block_number, mutable=False):
     return vm
 
 
+def _insert_transaction_to_pending(chain, transaction):
```

Comment:
Is the term `pending` here meant to refer to the "pending block"?

Reply:
Yes, I'll expand the name or add a docstring.

Reply:
The term *pending* is bugging me because of the collision with "pending transactions".

- A transaction added to the pending block **will** be mined when the block is mined.
- A transaction which is in the pending state, meaning it's in the transaction pool, doesn't have the same guarantees.

Any ideas for different terminology?

Reply:
I just posted a commit with it renamed to `_insert_transaction_to_pending_block`. Is that sufficient clarity? This is intended to be the first case, so that the transaction will be mined in the next block.

Reply:
That is sufficiently descriptive for me.